src/egon/data/datasets/fix_ehv_subnetworks.py

```diff
@@ -113,15 +118,26 @@ def add_line(x0, y0, x1, y1, v_nom, scn_name, cables):
     if v_nom == 220:
         s_nom = 520
         x_per_km = 0.001 * 2 * np.pi * 50
+        r_per_km = 0.097  # based on average r from similar lines
```

Comment:
But actually if it is a length-specific value and only for a 220kV overhead line, all of the lines of this type should have the same value, right? So if you take average should be the same as taking any value of this group? So value-wise it should not make a difference, I guess. It just reads a bit like as it would be just a rough estimate...

Reply:
I also thought that all the lines with the same capacity and voltage level should have the same resistance/km but it is not the case. I will proceed to write a more generic calculation using the values [here](https://github.com/wupperinst/osmTGmod/blob/ce70ba30b2b101cfccde9cc9d6fff3f0e5a12083/code/electrical_properties.sql#L48-L59).